app/services/database_health_checker.rb

```diff
@@ -0,0 +1,26 @@
+module DatabaseHealthChecker
+  module_function
+
+  Summary = Struct.new(:healthy, :result) do
+    def as_json(*args)
+      to_h.as_json(*args)
+    end
+  end
+
+  # @return [Summary]
+  def check
+    Summary.new(true, simple_query)
+  rescue => err
+    Summary.new(false, err.message)
```

Comment:
I think for the time being, this will be safe? But happy to clean it up later if we find we're leaking stuff we shouldn't.